### PR TITLE
Translation to Italian and newlines in English instructions

### PR DESCRIPTION
**Pull request by soil8rino**
Added Italian translations of ingredients, names and directions
Fixed some typos (colling instead of collins) in English directions
Added \n to  in English directions were necessary
Corrected the amount of pina colada ingredients

### CODE REVIEW

File: src/locales/it/istructions.json
Comment (anton-gustafsson) [inline src/locales/it/istructions.json:1]: typo. It should be **instructions.json**

**Comment by anton-gustafsson** [comment]
Thank you once again! 🚀 

Would you mind fixing comment above about a typo and also update the name of the PR

**Comment by soil8rino** [comment]
I corrected the typo, but I do not know how to update the name of the PR. I will undoubtedly do it if you explain how

**Comment by anton-gustafsson** [comment]
I think you can just click on the title here on github and change it

**Comment by soil8rino** [comment]
Is that what you wanted ?

**Comment by anton-gustafsson** [comment]
Yes, thanks!

**Pull request merged**